task8/index.js

```diff
@@ -0,0 +1,49 @@
+const obj = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"}, 
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+]
+
+const div = document.getElementById("js-parent");
+
+const loading = () => {
+    const loadingPlace = document.createElement("div");
+    const gif = document.createElement('img');
+    loadingPlace.id = "loadingPlace";
+    gif.src = "loading-circle.gif";
+    div.appendChild(loadingPlace).appendChild(gif);
+};
+
+const loaded = () => {
+    const loadingPlace = document.getElementById("loadingPlace");
+    loadingPlace.remove();
+}
+    
+const getData  = new Promise((resolve, reject) => {
+    loading();
+        setTimeout(() => {
+        reject(obj);
+    },3000);
+});
+
+getData.then((value) => {
+    loaded();
+    const ul = document.createElement('ul');
+    const fragment = document.createDocumentFragment();
+
+    value.forEach(item => {
+        const li = document.createElement('li');
+        const a = document.createElement('a');
+        const img = document.createElement('img');
+
+        a.textContent = item.text;
+        a.href = item.to;
+        img.src = item.img;
+        img.alt = item.alt;
+
+        fragment.appendChild(li).appendChild(a).appendChild(img);
+    })
+    div.appendChild(ul).appendChild(fragment);
+});
+getData.catch(() => {
+    console.log("error!");
+});
```

Comment:
This is a continuation of what I commented above.
The string `error!` has now been passed as an argument to reject.

So if you output the value passed as an argument to the console, it will work as per the specification of this issue.
By the way, if you want to output errors to the console, use
If you want to output errors to the console, it is better to use the API `console.error`.
There are also various other APIs for console.[ Please take a look at them if you like](https://www.nxworld.net/js-console-api.html).

Reply:
I only knew about the console logs! 
Thanks for sharing something so interesting!